src/containers/Filter.js

```diff
@@ -0,0 +1,35 @@
+import React from "react"
+
+export default class FilterContainer extends React.Component {
+  onFilterChange = (e) => {
+    const {target: {value}} = e
+    this.props.layoutActions.updateFilter(value)
+  }
+
+  render () {
+    const {specSelectors, layoutSelectors, getComponent} = this.props
+    const Col = getComponent("Col")
+
+    const isLoading = specSelectors.loadingStatus() === "loading"
+    const isFailed = specSelectors.loadingStatus() === "failed"
+    const filter = layoutSelectors.currentFilter()
+
+    const inputStyle = {}
+    if (isFailed) inputStyle.color = "red"
+    if (isLoading) inputStyle.color = "#aaa"
+
+    return (
+      <div>
+        {filter === null || filter === false ? null :
+          <div className="filter-container">
+            <Col className="filter wrapper" mobile={12}>
+              <input className="operation-filter-input" aria-label="Input for filtering by tag" placeholder="Filter by tag" type="text"
+                     onChange={this.onFilterChange} value={filter === true || filter === "true" ? "" : filter}
+                     disabled={isLoading} style={inputStyle}/>
```

Comment:
Typically, I'd prefer a `<label>` for a text input over `aria-label`, but I'm guessing that would significantly alter the UI, yeah?

Reply:
<img width="230" alt="Screen Shot 2022-04-06 at 1 33 22 PM" src="https://user-images.githubusercontent.com/40131297/162054645-8864aa2b-5dcb-4d5f-97ca-328025e1185b.png">
It's this input right here 🤔 I could add a label above it if that's okay from a design perspective. I think it might be safer to just have the `aria-label` in case this might impact user's layouts. What do you think?

Reply:
@codelemur PING as I forgot to ping you :D

Reply:
lol np

Reply:
Yeah @davidma415 I think it should be fine to leave with `aria-label`